File: 2024-2025/game.cs
using System;

namespace TextAdventure
{
    class Program
    {
        static void Main(string[] args)
        {
            Console.WriteLine("You wake up in a dark and damp cellar. Your flashlight has almost dead batteries, but you can see a staircase going up, and you can see 3 doors marked A, B, and C.");
            Console.WriteLine("You hear a terrible noise upstairs.");
            Console.WriteLine("What do you choose?");
            Console.WriteLine("A: Open door A");
            Console.WriteLine("B: Open door B");
            Console.WriteLine("C: Open door C");
            Console.WriteLine("T: Go up the stairs");

            string choice = Console.ReadLine();

            switch (choice.ToUpper())
            {
                case "A":
                    OpenDoorA();
                    break;
                case "B":
                    OpenDoorB();
                    break;
                case "C":
                    OpenDoorC();
                    break;
                case "T":
                    GoUpstairs();
                    break;
                default:
                    Console.WriteLine("Invalid choice. Try again.");
                    break;
            }
        }

        static void OpenDoorA()
        {
            Console.WriteLine("You open door A and find a treasure chest filled with gold.");
            // Add more logic here
        }

        static void OpenDoorB()
        {
            Console.WriteLine("You open door B and find a sleeping dragon. Better close it quietly!");
            // Add more logic here
        }

        static void OpenDoorC()
        {
            Console.WriteLine("You open door C and find a passage to the outside world.");
            // Add more logic here
        }

        static void GoUpstairs()
        {
            Console.WriteLine("You go up the stairs and find yourself in a haunted house.");
            // Add more logic here
        }
    }
}

File: 2024/game.py
def main():
    print("You wake up in a dark and damp basement. Your flashlight has almost dead batteries, but you can still see a staircase going up, and you can see 3 doors marked A, B, and C.")
    print("You can hear a terrible noise upstairs.")
    print("What do you choose?")
    print("A: Open door A")
    print("B: Open door B")
    print("C: Open door C")
    print("T: Go up the stairs")

    choice = input().upper()

    if choice == "A":
        print("You open door A and find a hidden treasure!")
    elif choice == "B":
        print("You open door B and encounter a fierce monster!")
    elif choice == "C":
        print("You open door C and discover a secret passage!")
    elif choice == "T":
        print("You go up the stairs and escape to safety!")
    else:
        print("Invalid choice. Please choose A, B, C, or T.")

if __name__ == "__main__":
    main()
